### DISCUSSION TITLE
Wordmat is not installing

### DISCUSSION THREAD

**Discussion by MaxRolighed** [Eduap-com/WordMat#68] [Support] [answered]
Hello
I am using windows. My school got office. Lately i deleted everything from my computer and starting reinstalling everythingg. When i downloaded Office, I got the usual, word, excel, powerpoint and so on.  But when I installed wordmat, an error appeared. You can see it in the attached files. It translates to "You haven't installed Excel 2007, 2010, 2013 or 2016. Installation canceled. I've got Excel downloaded, and it works fine. I have even installed older versions of Wordmat, and also reinstalled the Office several times. I keep getting this error. Hope I can get some help :)

- Max (Student)

![wordmat virker ikke](https://user-images.githubusercontent.com/65622218/82365927-a913f380-9a11-11ea-8363-e84e0de02205.PNG)

**Comment by Glymse** [comment]
I had the same error, it believe it happens because word is not locally on your device drive. But its an microsoft app.

To fix this issue,
Go to your office 365 app, and you will find a button that says "_installer office_" / "_install office_", 
This should download all the word products to your device drive, and fix the install error

**Comment by Eduap-com** [comment]
Problemet rapporteres en gang i mellem. For at kunne løse problemet er jeg nødt til selv at støde ind i det, så jeg kan undersøge computeren.

**Comment by DirkOS13** [comment]
Update 1.24 Win10, Microsoft® Word til Microsoft 365 MSO (Version 2203 Build 16.0.15028.20218) 64-bit, install shows no presence of propper Word, like MaxRolighed. 
Apps are installed local, i can aktivate the macros, but its like WordMat is missing files 4 propper function...

**Comment by Zetikan** [comment, marked as answer]
Jeg havde en kursist hos os som havde problemet de sidste par dage. Jeg fik løst problemet ved at reparer Office pakken i tilføj/fjern programmer og reinstaller Wordmat bagefter.

**Comment by Eduap-com** [comment]
Ja, det kan nok løse problemet for nogle. Jeg har dog rapporter om nogle der har prøvet det uden held, så problemet er nok komplekst, og kan opstå på forskellige måder.

> Jeg havde en kursist hos os som havde problemet de sidste par dage. Jeg fik løst problemet ved at reparer Office pakken i tilføj/fjern programmer og reinstaller Wordmat bagefter.
